### PR TITLE
updating Observability support statement

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/26952#issuecomment-1423324813

### CODE REVIEW

File: observability/observability_enable.adoc [hunk 1]
```diff
@@ -25,8 +25,6 @@ Monitor the health of your managed clusters with the observability service (`mul
 * https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction[Azure storage]
 * https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation (formerly known as Red Hat OpenShift Container Storage)]
 * https://www.ibm.com/docs/en/baw/20.x?topic=storage-preparing-cloud-public-roks[Red Hat OpenShift on IBM (ROKS)]
-+
-*Important:* When you configure your object store, ensure that you meet the encryption requirements necessary when sensitive data is persisted. For more information on Thanos supported object stores, see https://thanos.io/tip/thanos/storage.md/#object-storage[Thanos documentation].
```
Comment (berenss) [inline observability/observability_enable.adoc:29]: I'd say KEEP it here in this section.

File: observability/observe_environments.adoc [hunk 2]
```diff
@@ -32,8 +32,6 @@ The observability service deploys an instance of Prometheus AlertManager, which
 - {product-title-short} supports the function of the multicluster observability operator on user-provided third-party object storage that is S3 API compatible.  
 
 - {product-title-short} use commercial, reasonable efforts to assist in the identification of the root cause. If a support ticket is raised and the root cause has been determined to be a result of the customer-provided S3 compatible object storage, then the issue must be resolved using the customer support channels.
-+
-*Important*: When you configure your object store, ensure that you meet the encryption requirements necessary when sensitive data is persisted. For a complete list of the supported object stores, see https://thanos.io/tip/thanos/storage.md/#object-storage[Thanos documentation].
```
Comment (berenss) [inline observability/observe_environments.adoc:35]: While we get rid of the note from this section, lets be mindful to add it back into 1.1.1. I want to make sure the user/reader knows that we use Thanos in our Observability service:

> The observability service deploys an instance of Prometheus AlertManager, which enables alerts to be forwarded with third-party applications. It also includes an instance of Grafana to enable data visualization with dashboards (static) or data exploration. Red Hat Advanced Cluster Management supports version 8.1.3 of Grafana. You can also design your Grafana dashboard. For more information, see [Designing your Grafana dashboard]. You can customize the observability service by creating custom recording rules or alerting rules. 
eg add: 
For more information about our centralized long term metric data storage, please see https://thanos.io/tip/thanos/storage.md/#object-storage[Thanos documentation].

**Review by berenss — CHANGES_REQUESTED** [review]
please include the changes into 1.1.1 to mention Thanos

File: observability/observability_enable.adoc [hunk 3]
```diff
@@ -26,7 +26,7 @@ Monitor the health of your managed clusters with the observability service (`mul
 * https://www.redhat.com/en/technologies/cloud-computing/openshift-data-foundation[Red Hat OpenShift Data Foundation (formerly known as Red Hat OpenShift Container Storage)]
 * https://www.ibm.com/docs/en/baw/20.x?topic=storage-preparing-cloud-public-roks[Red Hat OpenShift on IBM (ROKS)]
 +
-*Important:* When you configure your object store, ensure that you meet the encryption requirements necessary when sensitive data is persisted. For more information on Thanos supported object stores, see https://thanos.io/tip/thanos/storage.md/#object-storage[Thanos documentation].
+*Important:* When you configure your object store, ensure that you meet the encryption requirements necessary when sensitive data is persisted. The observability service uses Thanos supported, stable object stores.
```
Comment (cadawson) [inline observability/observability_enable.adoc]:
```suggestion
*Important:* When you configure your object store, ensure that you meet the encryption requirements that are necessary when sensitive data is persisted. The observability service uses Thanos supported, stable object stores.
```

**Review by cadawson — APPROVED** [review]
One suggestion. Looks good! Thanks!

**Review by berenss — APPROVED** [review]
looks good thank you!

**Pull request merged**